app/extensions.js

```diff
@@ -197,6 +198,13 @@ module.exports.init = () => {
   let enableExtensions = () => {
     installExtension(config.braveExtensionId, getExtensionsPath('brave'), {manifest_location: 'component', manifest: generateBraveManifest()})
 
+    if (getSetting(settings.PDFJS_ENABLED)) {
```

Comment:
PDF.js is consistently slower than just downloading the file and opening it in Preview, so some people probably won't like it even though it's built in.